server/src/lobby.rs

```diff
@@ -631,6 +651,14 @@ mod name_validation {
         }
     }
 
+    pub fn sanitize_server_name(desired_name: String) -> String {
+        desired_name
+            .remove_newline()
+            .trim_whitespace()
+            .truncate(MAX_SERVER_NAME_LENGTH)
+            .truncate_lines(1)
+    }
+
```

Comment:
why not replace this function with just
sanitize_text(desired: String, length: usize) or something similar

Reply:
I decided against this. Not all text should be truncated to one line, so having this be a generalized `sanitize_text` doesn't make all that much sense.
Plus, there's not much point to generalizing this. We don't need to sanitize that much text, especially not in this specific way.